contracts/contracts/FleekERC721.sol

```diff
@@ -116,7 +144,8 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
         string memory commitHash,
         string memory gitRepository,
         string memory logo,
-        uint24 color
+        uint24 color,
+        bool accessPointAutoApprovalSettings // defaults to `false` if not specifically passed as `true`
```

Comment:
Seems like this has no default value as it is a required param.

Reply:
EVM considers the default value to be false automatically. No need to specify.